best_practices/deploy/deploy

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+#############MANDATORY#########
+PROJECT_NAME=
+###############################
+
+if [ -z "$DOCKERHUB_USER" ]; then
+  DOCKERHUB_USER='codelittinc'
+fi
+
+if [ -z "$DEPLOY_TYPE" ]; then
+  exit "You need to pass the deploy type(DEPLOY_TYPE), example: qa, production, beta e etc."
```

Comment:
`exit "You need to pass the deploy type(DEPLOY_TYPE), example: qa, production, beta e etc."`

Should not have extra "e" before "etc". Just a comma -> ","